config/campaignSets.json

```diff
@@ -5,13 +5,13 @@
 		"exitbutton" : {"x": 658, "y": 482, "name":"CMPSCAN" },
 		"items":
 		[
-			{ "x":90,  "y":72,  "file":"DATA/GOOD1.H3C",    "image":"CAMPGD1S", "video":"CGOOD1",   "open": true },
-			{ "x":539, "y":72,  "file":"DATA/EVIL1.H3C",    "image":"CAMPEV1S", "video":"CEVIL1",   "open": true },
-			{ "x":43,  "y":245, "file":"DATA/GOOD2.H3C",    "image":"CAMPGD2S", "video":"CGOOD2",   "open": true },
-			{ "x":313, "y":244, "file":"DATA/NEUTRAL1.H3C",  "image":"CAMPNEUS", "video":"CNEUTRAL", "open": true },
-			{ "x":586, "y":246, "file":"DATA/EVIL2.H3C",    "image":"CAMPEV2S", "video":"CEVIL2",   "open": true },
-			{ "x":34,  "y":417, "file":"DATA/GOOD3.H3C",    "image":"CAMPGD3S", "video":"CGOOD3",   "open": true },
-			{ "x":404, "y":414, "file":"DATA/SECRET1.H3C",   "image":"CAMPSCTS", "video":"CSECRET",  "open": true }
+			{ "id": 1, "x":90,  "y":72,  "file":"DATA/GOOD1",    "image":"CAMPGD1S", "video":"CGOOD1",   "requires": []        },
+			{ "id": 2, "x":539, "y":72,  "file":"DATA/EVIL1",    "image":"CAMPEV1S", "video":"CEVIL1",   "requires": []        },
+			{ "id": 3, "x":43,  "y":245, "file":"DATA/GOOD2",    "image":"CAMPGD2S", "video":"CGOOD2",   "requires": [1, 2, 4] },
+			{ "id": 4, "x":313, "y":244, "file":"DATA/NEUTRAL1", "image":"CAMPNEUS", "video":"CNEUTRAL", "requires": []        },
+			{ "id": 5, "x":586, "y":246, "file":"DATA/EVIL2",    "image":"CAMPEV2S", "video":"CEVIL2",   "requires": [1, 2, 4] },
```

Comment:
Perhaps use filenames for requirements instead of integers? e.g. "DATA/GOOD1"?

Reply:
I like the current approach much more. For me it's cleaner.